pom.xml

```diff
@@ -107,10 +111,52 @@
                 <artifactId>junit-jupiter-engine</artifactId>
                 <version>${junit.jupiter.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.reactivestreams</groupId>
+                <artifactId>reactive-streams</artifactId>
+                <version>${reactive.streams.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.projectreactor</groupId>
+                <artifactId>reactor-test</artifactId>
+                <version>${projectreactor.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.reactivestreams</groupId>
+                <artifactId>reactive-streams-tck</artifactId>
+                <version>${reactive.streams.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
```

Comment:
Is there a specific reason to add testNG next to Junit?

Reply:
Ok, noticed later on that it's used to re-use Reactive Stream's test suites for publishers. Maybe nice to document this?

Reply:
Done